jmclient/jmclient/blockchaininterface.py

```diff
@@ -769,11 +781,17 @@ def query_utxo_set(self, txout, includeconf=False):
         return result
 
     def estimate_fee_per_kb(self, N):
+        if super(BitcoinCoreInterface, self).fee_per_kb_has_been_manually_set(N):
+            return N
         estimate = int(Decimal(1e8) * Decimal(self.rpc('estimatefee', [N])))
+        if (N == 1) and (estimate < 0):
+            # Special bitcoin core case: sometimes the highest priority
+            # cannot be estimated in that case the 2nd highest priority
+            # should be used instead of falling back to hardcoded values
+            estimate = Decimal(1e8) * Decimal(self.rpc('estimatefee', [N+1]))
```

Comment:
I have a memory that a non-integer value being returned craps something out. Can you check that?